NDB.Covid19/NDB.Covid19/Locales/nb.json

```diff
@@ -235,7 +235,7 @@
   "NOTIFICATION_ANDROID_UPDATE_DESCRIPTION": "Gå til Google Play Butikk for å oppdatere",
   "NOTIFICATION_CONSENT_HEADER": "Det er viktig at du åpner appen og godkjenner samtykket på nytt.",
   "NOTIFICATION_BACKGROUND_FETCH_HEADER": "Smittestopp henter informasjon",
-  "NOTIFICATION_BACKGROUND_FETCH_DESCRIPTION": "Smittestopp undersøker om du vær i nærheten av brukere som har meldt seg smittet.",
+  "NOTIFICATION_BACKGROUND_FETCH_DESCRIPTION": "Smittestopp undersøker om du var i nærheten av brukere som har meldt seg smittet.",
```

Comment:
Jeg synes "har vært" høres bedre ut

Reply:
We should deploy this ASAP IMO, because that is what people are seeing in the foreground notification.

Reply:
I have seen this line in the app too, and I'm glad there is already a PR on it.
I agree that "har vært" is probably correct here.

Reply:
Enig, en liten brainfart fra meg der.